pipelines/respondent-home-ui.yml

```diff
@@ -256,6 +296,15 @@ jobs:
     file: ras-deploy/tasks/extract-source-from-release.yml
     input_mapping: { release-resource: respondent-home-ui-release }
     output_mapping: { release-source: respondent-home-ui-release-source }
+  - put: create-redis
```

Comment:
There's an existing instance of redis in pre-prod as well so might not need to create an instance here? It might just fail gracefully anyway

Reply:
I expect it's useful to have create-redis in each of the pipelines that depend upon it in case there is a situation where it has not yet been provisioned for any.